### DISCUSSION TITLE
Module not found: Can't resolve 'dgram' from ./node_modules/dns/node_modules/native-dns/lib/server.js

### DISCUSSION THREAD

**Discussion by samueloshio** [nextauthjs/next-auth#8341] [Help]
Am stuck while trying to use Email Provider. Have gotten couple of similar issues where most of these modules are depreciated already;
Before now i had Module not found: Can't resolve 'fs' and Module not found: Can't resolve 'dns' both coming from /node_modules/nodemailer. what i did to resolve these issues was to add fs: false in the next.config.js while i run npm i net for the latter. Now am completely stuck at "dgram" module not found.

**Comment by nicitaacom** [comment]
Same for me

I'm developing outreach-tool using redis (ioredis)
when I'm firing

```ts
const email = new Emails()
```

it throwing the error 'dns' 'net' 'tls' 'dgram' not found - this errors possible to fix by `pnpm add dns net tls dgram` etc but I don't want to install +100500 dependencies


somewhere in Emails.ts (this is `export class Emails`)
```ts
 async getCachedScheduledEmailObjects(
    something: string[] | string,
    somethingElse: string,
  ): Promise<TScheduledEmail[] | string> {
  return ''

  }

```
